components/LocaleSwitcher/LocaleSwitcher.vue

```diff
@@ -71,38 +71,44 @@
     </ul>
     <span class="separator" />
   </div>
-  <div v-else-if="type === 'only-text'">
-    <ul class="locale-switcher-burger-menu">
+  <div v-else-if="type === 'only-text'" class="locale-switcher-burger-menu">
+    <p class="locale-switcher-burger-menu__title">
+      {{ $t('locale-switcher.title') }}
+    </p>
+    <ul class="locale-switcher-burger-menu__list">
       <li v-for="option in menu" :key="option.key">
-        <template v-if="option.children">
-          <span
+        <ul v-if="option.children">
+          <li
             v-for="child in option.children"
             :key="child.name"
             :class="{
```

Comment:
Suggestion enlever le : true après la classe 😄 
```suggestion
            :class="[
              'locale-switcher-burger-menu-list__locale',
              {
                'locale-switcher-burger-menu__list--active':
                  child.localeCode === currentLocaleCode,
              },
            ]"
```